packages/ui/src/machines/authenticator/signUp.ts

```diff
@@ -201,6 +226,15 @@ export function createSignUpMachine({ services }: SignUpMachineOptions) {
         setUser,
       },
       services: {
+        async signInService(context, event) {
```

Comment:
one more naming nit:
```suggestion
        async signIn(context, event) {
```

Reply:
Will do!